samples.json

```diff
@@ -58,5 +58,13 @@
             "baseTemplateSupport": "Toradex",
             "sampleSupport": "Community"
         }
+    },
+    "VPU": {
+        "generic": {
+            "folder": "vpuGeneric",
+            "baseTemplate": "genericTemplate",
+            "baseTemplateSupport": "Toradex",
+            "sampleSupport": "Community"
+        }
```

Comment:
Please check If I added the correct values here.

Reply:
Not sure what criteria @andreriesco is using for the `sampleSupport`, but I see he used `Community` on https://github.com/toradex/torizon-samples/commit/62e2c905af211327168af3f68ca055d2a71ecb73 so sounds feasible. Let's see what are his thoughts.

Reply:
There is no formal defined criteria about this. The difference about a Toradex maintained and a Community maintained one is that on the Toradex one we ensure that we Toradex will maintain the template and try to ensure it's quality. But since this can be considered a  bit experimental, like the tflite one, I think it os ok to have it as Community supported

Reply:
Sounds good to use Community then.